### PR TITLE
LPS-162024 | Test Automation | Master

### PR DESCRIPTION
**Pull request by JhossephG**
https://issues.liferay.com/browse/LPS-162024

### CODE REVIEW

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path [hunk 1]
```diff
@@ -580,6 +580,11 @@
 		<td>//div[@class='mt-1']</td>
 		<td></td>
 	</tr>
+	<tr>
+	<td>ERROR_MESSAGE_INPUT_DOES_NOT_MATCH</td>
+	<td>//div[contains(@class,'form-feedback-item')]</td>
+	<td></td>
+	</tr>
```
Comment (rafaellajordao) [inline modules/apps/object/object-test/src/testFunctional/paths/ObjectAdmin.path:587]: Hello Jhosseph, this path already exists in other places on the portal! You can replace it with _Message#WARNING_FEEDBACK_ :)

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 2]
```diff
@@ -323,6 +323,47 @@ definition {
...
+
+			SelectFrame(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+			Type(
+				locator1 = "ObjectAdmin#PLACEHOLDER_DELETE_OBJECT",
+				value1 = "anyone");
+		}
+
+		task ("Then the error message is showed on the modal") {
+			AssertElementPresent(locator1 = "ObjectAdmin#ERROR_MESSAGE_INPUT_DOES_NOT_MATCH");
```
Comment (rafaellajordao) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:363]: You can replace it with Message#WARNING_FEEDBACK :)

**Review by rafaellajordao — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase (hunk 2 above)
Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:363]: Done, thanks! ;)

**Comment by JhossephG** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 3]
```diff
@@ -323,6 +323,47 @@ definition {
 			locator1 = "CreateObject#SELECT_OBJECT_FIELD_IN_TABLE");
 	}
 
+	@description = "LPS-162024 - Verify that an error message is shown when the user enters the wrong value in the confirmation field"
+	@priority = "4"
+	test CanViewErrorMessageWithIncorrectValue {
+		task ("Given an object with an attachment field is created") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:329]: "Given an object with a field is created"

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 4]
```diff
@@ -323,6 +323,47 @@ definition {
...
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "false",
+				objectName = "CustomObject162024");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject162024");
+		}
+
+		task ("When try delete the object with a incorrect value") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:346]: "When the user enters an incorrect value in the confirmation field while deleting the object"

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase [hunk 5]
```diff
@@ -323,6 +323,47 @@ definition {
...
+			MenuItem.clickNoError(menuItem = "Delete");
+
+			SelectFrame(locator1 = "ObjectCustomViews#VIEW_MODAL_CONTENT");
+
+			Type(
+				locator1 = "ObjectAdmin#PLACEHOLDER_DELETE_OBJECT",
+				value1 = "anyone");
+		}
+
+		task ("Then the error message is showed on the modal") {
```
Comment (epark18) [inline modules/apps/object/object-test/src/testFunctional/tests/Object.testcase:362]: "Then an error message is shown"

**Review by epark18 — COMMENTED** [review]
@JhossephG This is good, I just left some suggestions on the wording of task blocks. Sometimes the ticket will have it written out, so you're always welcome to use those.

File: modules/apps/object/object-test/src/testFunctional/tests/Object.testcase (hunk 5 above)
Comment (JhossephG) [reply modules/apps/object/object-test/src/testFunctional/tests/Object.testcase]: Everything was done! :)

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
Great job @JhossephG! This looks good, but it looks like there will be merge conflicts. Please rebase when you can. Thank you!

**Comment by JhossephG** [comment]
> Great job @JhossephG! This looks good, but it looks like there will be merge conflicts. Please rebase when you can. Thank you!

Thanks, Eric! Was done!

**Comment by JhossephG** [comment]
ci:test:sf

**Comment by epark18** [comment]
ci:forward

**Pull request closed without merging**